File: calculator.py
def main():
    x = int(input("Valor de x:"))
    y = int(input("Valor de y:"))
    z = input("escolha a operação a ser realizada:('+','-','/','*'\n)")
    resultado = operador(x,y,z)
    print(f"Total:{resultado}")

def operador (x,y,z):
    if z == '+' :
        return x + y
    elif z == '-' :
        return x - y
    elif z == '*' :
        return x * y
    elif z == '/' :
        return x / y
    else:
        print("Digite um operador Válido!")
        return None
main()

File: calculator.c
#include <cs50.h>
#include <stdio.h>

float add(float a, float b, char operador);

int main (void)
{
    
    float x = get_float ("Insira o valor de x:\n");
    char z = get_char ("Insira a Operação: (+,-,/,*)\n");
    float y = get_float ("Insira o valor de y\n"); 
    float c = add(x,y,z);
    printf ("Resultado : %.2f\n",c);
}
    float add(float a, float b,char operador){
     
     if (operador == '+') {    
        return a + b ;
     
    }
    else if (operador == '-') {
        return a - b;
       
    }
    else if (operador =='/') { 
        return a / b;
     
    }
    else if (operador =='*'){
        return a * b;
    }
    else {
            printf ("Operação Invalida!");
    }
  
    }
